packages/iov-crypto/src/englishmnemonic.spec.ts

```diff
@@ -1,6 +1,36 @@
+import { Encoding } from "@iov/encoding";
+
 import { EnglishMnemonic } from "./englishmnemonic";
+import { Sha256 } from "./sha";
+import wordlists from "./testdata/bip39_wordlists.json";
+
+const { fromAscii, fromHex } = Encoding;
 
 describe("EnglishMnemonic", () => {
+  describe("wordlist", () => {
+    it("matches the words from the bitcoin/bips/bip-0039 spec", () => {
+      const lineFeed = 0x0a;
+      const bip39EnglishTxt = fromHex(wordlists["english.txt"]);
```

Comment:
Could we change the key to just `english`? Also why is this hex encoded?

Reply:
We could. I used english.txt to make explicit that this is english.txt (from https://github.com/bitcoin/bips/tree/master/bip-0039)

If we want test data that would equally well in browsers and nodejs, I think we need to use json, which we can import as a module. nodejs will read it from disk and webpack will inline it. But correct me of I'm wrong.

The choice between hex or base64 encoding was random.

What do you think?

Reply:
I think if conciseness is something to prioritise then base64 would be better. But what's wrong with utf8? Since this issue is about making sure we have the correct wordlist, anything that's not human-readable is going to make it more difficult to notice if we've got out of sync with the spec somehow.

Reply:
The point is I want to store and process binary data and avoid any kind of string encoding between english.txt and `bip39EnglishTxt`. As soon as we use a JSON string, we receive a string in JavaScript that involves a bunch of implicit conversion logic (UTF8 source code to unicode string). I cannot test the checksum of a string directly.

I'm updating to `.english` and base64 now, The additional comment is coming too.